src/helpers/MSFS/parseCFGFile.js

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs';
+import ini from '../Manifest/CFGFileParser.js';
+import ThrowError from '../ThrowError';
+
+let CFGpath = "";
+const pathRegEx = /^[a-z]:((\\|\/)[a-z0-9\s_@\-^!#$%&+={}\[\]]+)+\.cfg$/i;
+
+/**
+ * Set path to a CFG file
+ *
+ * @export
+ * @argument {String} path  Set path to CFG file
+ */
+
+export function loadCFG(path) {
+    CFGpath = path;
+    console.log(path)
+}
+/**
+ * Parse chosen CFG file and return it
+ *
+ * @export
+ * @return {Object} JSON object with the CFG file data
+ */
+
+export function parseCFGFile() {
+    if (!pathRegEx.test(CFGpath) || !fs.lstatSync(CFGpath).isFile()) return ThrowError('E007', `Invalid path ${CFGpath}`);
+    const data = ini.parse(fs.readFileSync(CFGpath, 'utf8'));
+    console.log(data);
```

Comment:
Sure. Just used this to test everything, I think it is a good function since you don't need to write the parse and check path stuff each and everytime ;)